File: src/main/java/com/connectflow/dto/CreateUserRequest.java
package com.connectflow.dto;

import com.connectflow.model.UserRole;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CreateUserRequest {
    private String fullName;
    private String email;
    private String phone;
    private String password;
    private UserRole.Role role;
    private UUID branchId;
}

File: backend/src/dto/CreateUserRequest.js
'use strict';
class CreateUserRequest {
  constructor({ fullName, email, phone, password, role, branchId }) {
    this.fullName = fullName;
    this.email = email;
    this.phone = phone;
    this.password = password;
    this.role = role;
    this.branchId = branchId;
  }
}
module.exports = CreateUserRequest;
